build/ndm-daemonset/entrypoint.sh

```diff
@@ -4,7 +4,10 @@ export GOTRACEBACK=crash
 
 echo "[entrypoint.sh] enabling core dump."
 ulimit -c unlimited
-echo "/var/openebs/sparse/core.%e.%p.%h.%t" > /proc/sys/kernel/core_pattern
+echo "[entrypoint.sh] creating /var/openebs/sparse if not exists."
+mkdir -p /var/openebs/sparse
+echo "[entrypoint.sh] changing directory to /var/openebs/sparse"
+cd /var/openebs/sparse || exit
```

Comment:
will it work if SPARSE_FILE_DIR env is different than /var/openebs/sparse? should we use SPARSE_FILE_DIR env instead /var/openebs/sparse?

Reply:
Only the mountPath is dependent. `SPARSE_FILE_DIR` is the directory inside the container .

Reply:
right, so if I have changed that path to something else what will happen? since in the entrypint.sh it is fixed to /var/openebs/sparse.

Reply:
if the mountPath is also changed, the core dump wont be persisted.  Will raise a PR to fix it. The `SPARSE_FILE_DIR` and mountPath should always be the same.